app/views/request-to-go-live/organisation-name.njk

```diff
@@ -0,0 +1,28 @@
+{% extends "../layout.njk" %}
+{% from "components/input/macro.njk" import govukInput %}
+{% from "components/button/macro.njk" import govukButton %}
+{% from "components/error-summary/macro.njk" import govukErrorSummary %}
+
+{% block pageTitle %}
+  Request to go live - {{ currentService.name }} - GOV.UK Pay
+{% endblock %}
+
+{% block mainContent %}
+  <div class="govuk-grid-column-two-thirds govuk-!-margin-top-6">
+    <span id="request-to-go-live-current-step" class="govuk-caption-l">Step 1 of 3</span>
+    <h1 class="govuk-heading-l">What is the name of your organisation?</h1>
+
+    <form id="request-to-go-live-organisation-name-form" method="post" action="/service/{{currentService.externalId}}/request-to-go-live/organisation-name">
+      <input id="csrf" name="csrfToken" type="hidden" value="{{ csrf }}"/>
+      {{ govukInput({
```

Comment:
Inputs need to have a label (for accessibility), your `h1` at line 13 is it I guess.

Reply:
You could do it like
```html
label: {
    html: '
    <span id="request-to-go-live-current-step" class="govuk-caption-l">Step 1 of 3</span>
    <h1 class="govuk-heading-l">What is the name of your organisation?</h1>'
  },
```

But i dunno if this is dodgy, basically the page title is also the label. There is a pattern for this with guidance, will find it now.

Reply:
Found it https://design-system.service.gov.uk/patterns/question-pages/#label-as-a-page-heading, so there is a boolean `isPageHeading: true`

Reply:
Yes, we are copying what is in the mockup, looks like mockup does not treat the label as h1, but this looks better